### PR TITLE
Made with Code landing page

### LINKED ISSUE #1425: Create a Made with Code activities landing page
We currently host three Made With Code activities:
 * [Meme](https://teach.mozilla.org/activities/madewithcode-meme/)
 * [Movie Poster](https://teach.mozilla.org/activities/madewithcode-poster/)
 * [First Web Page](https://teach.mozilla.org/activities/madewithcode-firstwebpage/)

We need to give them a landing page, with links to all three.

Can use styles from https://teach.mozilla.org/activities/web-lit-basics/

url: /activities/madewithcode

Title: Made with Code

Image: use Made with Code logo and Google logo - see #1423 

Intro copy: Welcome! Mozilla is happy to partner with Google's Made with Code initiative to inspire girls to get creative with code. Below are three fun activities to help you create your own webpages by writing and remixing HTML.

[No "Learning Objectives" section"]

[No section headings - e.g. "Reading the Web"]

**Let's Make a Meme**
*Understanding remixing*
A “meme” is something that’s shared on the internet, and it usually has three parts: a picture, a caption, and a dash of humor. In this activity, you’ll make your own meme using a bit of HTML and your own creativity. 

**Make a Movie Poster**
*Understanding remixing*
Seen a movie or read any good books lately? Make a web page with your glowing (or vicious) review. Learn a little code along the way.

**Make Your First Webpage**
*Understanding remixing*
You’ve probably seen hundreds (if not thousands) of web pages. All of them are written in a language called HTML. It’s easy to start messing around with HTML. In this activity, you’ll make a basic web page in about 60 seconds or less.

Looking for more fun projects to make and remix using Mozilla's Thimble code editor? Check out [Mozilla's Thimble](https://thimble.mozilla.org)!

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #1425

### CODE REVIEW

File: pages/madewithcode.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+var Intro = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <h1>Made with Code</h1>
+        <section className="intro">
+          <Illustration
+          width={200} height={355}
+          src1x="/img/pages/madewithcode/logo_mwc.png"
```
Comment (Pomax) [inline pages/madewithcode.jsx]: I remember you saying if there was no x2 to just use "src", is that still true?

File: pages/madewithcode.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          alt="Woman training a young man on a computer"
+          className="content-first">
+            <h2>Welcome! Mozilla is happy to partner with Google's Made with Code initiative to inspire girls to get creative with code. Below are three fun activities to help you create your own webpages by writing and remixing HTML.</h2>
```
Comment (Pomax) [inline pages/madewithcode.jsx]: quotes for "made with code"?

File: pages/activities.jsx [hunk 3]
```diff
@@ -258,6 +258,13 @@ var ActivitiesPage = React.createClass({
               link="https://stephguthrie.makes.org/thimble/ODU3ODAxMjE2/hacking-my-media-with-x-ray-goggles"
               developedBy="Steph Guthrie, Kim Wilkens, 2013 MozGirls"
               description="In this activity, learners create a remix of a Wikimedia page as they think critically about the intersection of gender, culture, technology and identity."/>
+            <ActivityKit
+              src1x="/img/pages/activities/img-activity-madewithcode.png"
+              title="Made with Code"
+              level="Beginner"
+              link="/activities/madewithcode"
+              developedBy="Mozilla for Google's Made with Code initiative"
```
Comment (Pomax) [inline pages/activities.jsx]: quotes for "Made with Code"?

File: pages/madewithcode.jsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  curriculum: [
+    {
+      title: "",
+      activities: [
+        {
+          title: "Let's Make a Meme",
+          image1x: "/img/pages/madewithcode/doge.jpg",
+          image2x: "/img/pages/madewithcode/doge@2x.jpg",
+          subtitle: "Understanding remixing",
+          description: "A “meme” is something that’s shared on the internet, and it usually has three parts: a picture, a caption, and a dash of humor. In this activity, you’ll make your own meme using a bit of HTML and your own creativity.",
```
Comment (Pomax) [inline pages/madewithcode.jsx]: did we ever make a decision on whether to use "pretty quotes" or plain quotes?

File: pages/madewithcode.jsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      ]
+    }
+  ],
+  render: function () {
+    return (
+      <div className="inner-container">
+        <Intro/>
+        {this.curriculum.map(function (section, key) {
+          return (
+            <ActivitySection title={section.title} key={key} activities={section.activities} />
```
Comment (Pomax) [inline pages/madewithcode.jsx]: don't use the array position as a key =( (just use the title, it's what uniquely identifies this thing, not its array position)

**Comment by Pomax** [comment]
A few nites, but my main concern is that we're linking to old thimble, which is getting shut down "soon". It feels like we should move those activities over to new thimble (which might require some tutorial rewriting) and then link to those new ones instead.

File: pages/madewithcode.jsx (hunk 4 above)
Comment (mmmavis) [inline pages/madewithcode.jsx]: nope, we are being inconsistent - I think we should stick with pretty quotes though

File: pages/madewithcode.jsx (hunk 5 above)
Comment (mmmavis) [inline pages/madewithcode.jsx]: nice catch

File: pages/madewithcode.jsx (hunk 2 above)
Comment (mmmavis) [inline pages/madewithcode.jsx]: @Pomax did you mean Made with Code should be wrapped in double quotes? ie. `“Made with Code”`

File: pages/madewithcode.jsx (hunk 1 above)
Comment (mmmavis) [inline pages/madewithcode.jsx]: I think we are still doing `src1x` (see [here](https://github.com/mmmavis/teach.mozilla.org/blob/3745a26e3dcb4216d1886d9ea852c6f9bb7d5971/components/illustration.jsx#L19)) and the underlying <ImageTag> will [map the path(s) to `src` and `srcSet`](https://github.com/mmmavis/teach.mozilla.org/blob/3745a26e3dcb4216d1886d9ea852c6f9bb7d5971/components/imagetag.jsx#L22-L23)

**Comment by mmmavis** [comment]
@Pomax I have some questions for you. See above plz :grin:

**Comment by mmmavis** [comment]
@hannahkane were you able to migrate those old Thimble makes to new Thimble? Is there anything I can help?

**Comment by mmmavis** [comment]
@hannahkane nvm, didn't see https://github.com/mozilla/teach.mozilla.org/issues/1426#issuecomment-161137222 until now

File: pages/madewithcode.jsx (hunk 2 above)
Comment (Pomax) [inline pages/madewithcode.jsx]: yeah, it reads a little funny without them

**Comment by Pomax** [comment]
orrrrrr+

File: pages/madewithcode.jsx (hunk 2 above)
Comment (hannahkane) [inline pages/madewithcode.jsx]: I don't think names of programs take quotes. I'd prefer to keep as is.

Comment (mmmavis) [inline pages/madewithcode.jsx]: I'm more leaning towards without quotes too :grin: I'll leave this text unchanged.

**Comment by mmmavis** [comment]
:tada:

**Pull request merged**